.dagger-ci/README.md

```diff
@@ -0,0 +1,32 @@
+# Testing with pytest
```

Comment:
Can you add the steps required to even get the correct required version of pytest and dagger for python installed? On my machine it seems that the later is really troublesome to install via pip correctly.

Reply:
Isn't `requirements.txt` enough?

Reply:
It pulled in a wrong version of platformdirs for me so I had to uninstall that and manually grab 2.6.2 for example. Also add the line to run `python3 -m pip install r requirements.txt` as I forgot that was needed.

Reply:
`platformdirs` 2.6.2? I am using 3.9.1